src/modules/projects/ExternalSubmissionsViewModal.tsx

```diff
@@ -0,0 +1,165 @@
+        customFieldKey,
+        selected.customDataElements
+      );
+
+      // if item has a picklist, convert value to PickListOption(s) so we can display the readable label
+      if (item.pickListOptions) {
+        if (Array.isArray(value)) {
+          submissionValues[key] = value.map((v) => getOptionValue(v, item));
+        } else {
+          submissionValues[key] = getOptionValue(value, item);
+        }
+      } else {
+        submissionValues[key] = value;
+      }
+    });
+    return submissionValues;
+  }, [selected, definition]);
+
+  const [notes, setNotes] = useState(selected.notes || '');
+  const [status, setStatus] = useState(
+    selected.status || ExternalFormSubmissionStatus.New
+  );
+  const [spam, setSpam] = useState(selected.spam || false);
+
+  const [updateSubmission, { loading, error }] =
+    useUpdateExternalFormSubmissionMutation({
+      variables: {
+        id: selected.id,
+        input: {
+          notes: notes,
+          status: status,
+          spam: spam,
+        } as ExternalFormSubmissionInput,
+      },
+      onCompleted: (data) => {
+        if (!data?.updateExternalFormSubmission?.errors?.length) {
+          onClose();
+        }
+      },
+    });
+
+  if (error) throw error;
+
+  const deleteButton = useMemo(
+    () =>
+      selected && (
+        <DeleteMutationButton<
+          DeleteExternalFormSubmissionMutation,
+          DeleteExternalFormSubmissionMutationVariables
+        >
+          queryDocument={DeleteExternalFormSubmissionDocument}
+          variables={{ id: selected.id }}
+          idPath={'deleteExternalFormSubmission.externalFormSubmission.id'}
+          recordName='Submission'
+          onSuccess={() => {
+            cache.evict({
+              id: `ExternalFormSubmission:${selected.id}`,
+            });
+            onClose();
+          }}
+          onlyIcon
+        />
+      ),
+    [onClose, selected]
+  );
+
+  return (
+    <CommonDialog open={!!selected && !!definition} fullWidth onClose={onClose}>
+      <DialogTitle>Review Submission</DialogTitle>
+      <DialogContent>
+        <Stack gap={2} sx={{ my: 2 }}>
+          <Card sx={{ p: 2 }}>
+            <DynamicView
+              values={submissionValues}
+              definition={definition.definition}
+            />
+          </Card>
+          <Stack gap={1}>
+            <TextInput
+              value={notes}
+              label='Reviewer Notes'
+              multiline
+              minRows={2}
+              onChange={(event) => setNotes(event.target.value)}
+            />
+            <LabeledCheckbox
+              value={status === ExternalFormSubmissionStatus.Reviewed}
+              label='Mark as Reviewed'
+              onChange={(_e, checked) => {
+                setStatus(
+                  checked
+                    ? ExternalFormSubmissionStatus.Reviewed
+                    : ExternalFormSubmissionStatus.New
+                );
+              }}
+            />
```

Comment:
Assuming that a future PR on the PIT branch will hide this checkbox for forms that have been reviewed.

If we go with my suggestion from https://github.com/greenriver/hmis-frontend/pull/936, we could replace it with something like `This form has been reviewed [and here is the Enrollment it generated, if applicable]`